aws/rust-runtime/aws-credential-types/src/lib.rs

```diff
@@ -3,6 +3,19 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#![warn(
+    missing_debug_implementations,
+    missing_docs,
+    rust_2018_idioms,
+    rustdoc::missing_crate_level_docs,
+    unreachable_pub
+)]
+
+//! `aws-credential-types` provides the items concerned with AWS SDK credentials including:
+//! * A trait for credential providers
+//! * An opaque struct representing credentials
+//! * Concrete implementations of credential caching
+
 //! AWS SDK Credentials
```

Comment:
You may want to take a look at how this looks rendered with `cargo doc`...
```suggestion
//! AWS SDK Credentials
//!
//! `aws-credential-types` provides the items concerned with AWS SDK credentials including:
//! * A trait for credentials providers
//! * An opaque struct representing credentials
//! * Concrete implementations of credentials caching
```